config.yaml

```diff
@@ -10,7 +11,7 @@ options:
       If unset, the manifests will use the image registry from the kube-control relation
 
       example)
-        juju config kubernetes-metrics-server image-registry='rocks.canonical.com:443/cdk'
+        juju config kubernetes-metrics-server image-registry='k8s.gcr.io'
```

Comment:
is this supposed to be an example of an empty registry config? 
```suggestion
        juju config kubernetes-metrics-server image-registry=''
```

Reply:
uh... yes you're right!

Reply:
yes, correct.  Let me fix that too